src/Microsoft.AspNet.Mvc.ViewFeatures/ViewResult.cs

```diff
@@ -79,6 +79,7 @@ public class ViewResult : ActionResult
                             actionContext = context,
                             result = this,
                             viewName = viewName,
+                            viewEngineType = viewEngine.GetType(),
```

Comment:
It's going to print `CompositeViewEngine` all the time since thats the wrapper. You'd need to log this separately or change `ViewEngineResult` to include the `IViewEngine` instance that found the view.